### ISSUE TITLE
New Rule Request - No server logging for Mirage

### ISSUE THREAD

**Issue by robert-j-webb** [ember-cli/eslint-plugin-ember#147]
Hi, I wrote a simple rule that checks to make sure we haven't left any mirage server logging in the test code. I understand that mirage is becoming very popular in the ember community, and that this repo might be the right place for a mirage linting rule. Should I instead make a repo just for mirage linting or should I make a PR for my rule?

**Comment by jbandura** [comment]
This is a very good question. I'd say even though ember-cli-mirage is a enormously popular addon in ember community, it's still not part of the framework itself, which would mean that we should probably keep it's linting rules in separate repo. I might be wrong though 🤔 Any thoughts @michalsnik @Turbo87?

**Comment by Turbo87** [comment]
> which would mean that we should probably keep it's linting rules in separate repo

I tend to agree, though I don't understand yet what this rule is actually warning about. @robert-j-webb can you give an example?

**Comment by robert-j-webb** [comment]
Sure. Often when debugging acceptance tests with mirage, server logging is turned on with this:

```js
server.logging = true
```

This code shouldn't be committed into the code base. If it's turned on, then it will clutter the test output and make it hard to read. It's similar to leaving a console.log statement in.

**Comment by Dhaulagiri** [comment]
@robert-j-webb did you ever publish this rule anywhere? We are having a similar need to keep this from getting committed as well.

**Comment by robert-j-webb** [comment]
@Dhaulagiri I remember writing it but I don't think I finished it. It's actually something I keep coming back to and if I ever get more time, I'll definitely write a PR for it. Thanks for reminding me about this.

**Comment by kellyselden** [comment]
Closing as not suitable for this repo as mentioned above, but look forward to a mirage linting project!
